### PR TITLE
Prep idr

### PR DESCRIPTION
**Pull request by jburel**
Replaces PR #3 
cc @pwalczysko 

The main question is  around the publication strategy
 * attach the zip to the tag
 * push to artifactory (current implementation)
 * both

### CODE REVIEW

File: .github/workflows/build.yml [hunk 1]
```diff
@@ -26,3 +26,12 @@ jobs:
         with:
           name: omero-ms-image-region ${{ matrix.java }}
           path: build/distributions/*
+      - name: Set ARTIFACTORY_URL
+        run: |
+          echo "ARTIFACTORY_URL=https://artifacts.openmicroscopy.org/artifactory/ome.releases" >> $GITHUB_ENV
+      - name: Publish
+        if: startsWith(github.ref, 'refs/tags') && github.repository_owner == 'ome' && matrix.java == 11
+        run: ./gradlew publish
```
Comment (sbesson) [inline .github/workflows/build.yml]: Is the goal to publish the JAR or the micro-service bundle? Should the groupId coordinates be updated to reflect the ownership of this fork?

**Review by sbesson — COMMENTED** [review]
From a quick investigation on the VMs, all the testing that happened lately on `test122` was done with a version of the micro-service derived from the upstream `omero-ms-image-region 0.8.7`. The latest pulls on the development branch of this fork include the changes from the latest `omero-ms-image-region 0.10.0` release and this PR adds the `OMEZarrReader 0.5.1` dependency used by IDR as well as a deployment step.

From a code review perspective, the dependency change look good from my side and should allow to build a bundle of the micro-service with the required Bio-Formats dependencies. One inline question about the intent of the deployment mechanism and the coordinates under which the artifact should be published.

In terms of functional testing, do you also anticipate the spin-up of a new idr-testing environment where this build can be deployed and tested?

File: .github/workflows/build.yml (hunk 1 above)
Comment (jburel) [reply .github/workflows/build.yml]: I started to work on https://github.com/ome/ansible-role-omero-ms-image-region/pull/4 afterwards, the zip is required so I will look into the option of attaching the zip to the tag instead

**Comment by jburel** [comment]
commit 59df744055f63ff37e18a5956521414d37e3f496 tested via personal account
see https://github.com/jburel/omero-ms-image-region/releases

**Review by sbesson — APPROVED** [review]

**Pull request merged**